recipe/activate.sh

```diff
@@ -6,20 +6,23 @@ ERROR=false
 [[ "@cross_target_platform@" == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
 [[ "@cross_target_platform@" == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
 
+CUDA_CFLAGS=""
+CUDA_LDFLAGS=""
 if [ "${CONDA_BUILD:-0}" = "1" ]; then
-    CUDA_INCLUDE_DIR="${PREFIX}/${targetsDir}/include"
+    CUDA_CFLAGS="${CUDA_CFLAGS} -I${PREFIX}/${targetsDir}/include"
+    CUDA_CFLAGS="${CUDA_CFLAGS} -I${BUILD_PREFIX}/${targetsDir}/include"
+    CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${BUILD_PREFIX}/${targetsDir}/lib/stubs"
     # Needed to fix cross compilation
     export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_FIND_ROOT_PATH=$PREFIX;$BUILD_PREFIX/$HOST/sysroot;$BUILD_PREFIX/${targetsDir}"
 else
-    CUDA_INCLUDE_DIR="${CONDA_PREFIX}/${targetsDir}/include"
+    CUDA_CFLAGS="${CUDA_CFLAGS} -I${CONDA_PREFIX}/${targetsDir}/include"
+    CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${CONDA_PREFIX}/${targetsDir}/lib/stubs"
```

Comment:
I recall that Isuru [mentioned](https://github.com/conda-forge/cuda-nvcc-feedstock/pull/13#discussion_r1198303915) that `CONDA_PREFIX` is `BUILD_PREFIX`, could it be the reason?

Reply:
Think Isuru may have been referring to within an active conda-build session

Outside of a conda-build session (just in a normal conda environment), `$CONDA_PREFIX` is the equivalent of `$PREFIX`. In fact this can be seen in the phase in the log. There's a line that says `CONDA_PREFIX=$PREFIX` (unfortunately it is hard to link to this line reliably 😞)